apps/meteor/app/lib/server/lib/notifyUsersOnMessage.js

```diff
@@ -177,15 +177,14 @@ export async function notifyUsersOnMessage(message, room) {
 		return message;
 	}
 
+	// Update all the room activity tracker fields
```

Comment:
The threads were made to not bother those who don't follow them, if we start updating lastmessage everyone will have their room modified, the behavior is correct